fabfile.py

```diff
@@ -387,19 +390,19 @@ def change_shell():
         run('git clone git://github.com/robbyrussell/oh-my-zsh.git ~/.oh-my-zsh')
     sudo('chsh %s -s /bin/zsh' % env.user)
 
-def setup_user():
+def setup_user(username=getpass.getuser()):
```

Comment:
I definitely don't want a function parameter initialized with some function. Just use the global variable env.user.